.releaserc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "branches": ["master"],
```

Comment:
Do we release from master? I think we release from develop for CVS so it is worth double checking with DevOps as this will not be the correct trigger